packages/flutter/lib/src/material/radio.dart

```diff
@@ -469,6 +485,10 @@ class _RadioPaintState extends State<_RadioPaint> {
         radioTheme.fillColor?.resolve(inactiveStates);
     final Color effectiveInactiveColor =
         inactiveColor ?? defaults.fillColor!.resolve(inactiveStates)!;
+    final Color activeBackgroundColor =
+        widget.backgroundColor?.resolve(activeStates) ?? Colors.transparent;
+    final Color inactiveBackgroundColor =
+        widget.backgroundColor?.resolve(inactiveStates) ?? Colors.transparent;
```

Comment:
I'm not sure how we want to handle those transparent colors? The other properties are using `_RadioDefaultsM3` or `_RadioDefaultsM2` as a fallbacks.

What is the best thing to do here?

Reply:
@QuncCccccc Is this something that could/should be in the Material token database? Or is this ok as-is?

Reply:
At some point the property will be in RadioTheme right? We should default to the property in RadioTheme instead

Reply:
sgtm as iterative change. consider add a todo here with the link to the issue for adding themedata so we don't miss this

Reply:
Makes sense, I added a TODO in [Add TODO](https://github.com/flutter/flutter/pull/169415/commits/343aca39c1170c206efeca03673f9d3150c5006c)